### DISCUSSION TITLE
Group articles list by feed/author?

### DISCUSSION THREAD

**Discussion by quinn-p-mchugh** [martinrotter/rssguard#1347] [Q&A] [answered]
Hi there,

RSS Guard is an amazing program - a huge thanks to all contributors!

Inoreader provides the ability to display articles grouped by feed, as shown below:
![Inoreader articles grouped by feed](https://github.com/martinrotter/rssguard/assets/40648753/b252cc64-2d30-435b-80a6-f3feb3197594)

Can RSS Guard do the same?

I'm aware you can quickly switch between feeds using `A` and `S` keys, but this is a bit jarring for me mentally. I'd be great if you could view a category and group the articles within that category by feed/author.

**Comment by quinn-p-mchugh** [comment, marked as answer]
I just realized I can simply sort by the `Author` column and achieve the kind of grouping I described above. 🤦

Even so, being able to group articles by feed/author in a tree view (like Inoreader does) might be a nice enhancement!

**Reply by martinrotter** [reply]
RSS Guard also supports multicolumn sorting, at this point up to 3 columns. Simply order by column, then click to order another column and all will be sorted by primary column, then if equaly by the next column etc. This is also remembered.

Implementing richer modern tree-like view is somewhere deep buried in my TODO.

**Reply by quinn-p-mchugh** [reply]
Nice! I didn't realize multi-column sorting was something you could do.

Thanks @martinrotter!
